easybuild/easyconfigs/i/ipyparallel/ipyparallel-6.2.2-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'Bundle'
+
+name = 'ipyparallel'
+version = '6.2.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ToFuProject/tofu'
+description = """Tomography for Fusion."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('IPython', '6.4.0', versionsuffix),
+]
+
+# raise an error when auto-downloading of dependency packages is detected
+exts_download_dep_fail = True
+
+exts_defaultclass = 'PythonPackage'
+exts_list = [
+    (name, version, {
```

Comment:
@SimonPinches If only one single Python package is involved, there's no need to use `Bundle`, just use `PythonPackage` directly (which significantly simpliies the easyconfig).

We should get this properly documented...